projects/android-identity-library.md

```diff
@@ -0,0 +1,148 @@
+      data is stored encrypted at rest.
+- Data structures and code for provisioning of mdoc/mDLs
+    - This code can can be used both on the device and issuer side. No networking
+      protocol is defined, the application has to define its own.
+- Parsers and generators for all data structures used in ISO/IEC 18013-5:2021
+  presentations, including `DeviceResponse`, `DeviceRequest`,  `MobileSecurityObject`
+  and many other CBOR data structures.
+- An implementation of the ISO/IEC 18013-5:2021 presentation flows including
+  QR engagement, NFC engagement (both static and negotiated), device retrieval
+  (BLE, Wifi Aware, and NFC)
+
+## Wallet and Reader Android applications
+
+This repository also contains two Android applications using this library
+in the `appholder` and `appverifier` modules. The Wallet application is a simple
+self-contained application which allows  creating a number of mdoc credentials
+using four different mdoc Document Types:
+
+- `org.iso.18013.5.1.mDL`: Mobile Driving License
+- `org.micov.1`: mdoc for eHealth ([link](https://github.com/18013-5/micov))
+- `nl.rdw.mekb.1`: mdoc for Vehicle Registration ([link](https://github.com/18013-5/mVR))
+- `eu.europa.ec.eudiw.pid.1`: mdoc for Personal Identification
+
+and their associated mdoc name spaces. The first one is defined in
+ISO/IEC 18013-5:2021 and the other three have been used at mdoc/mDL
+test events organized by participants of the ISO/IEC JTC1 SC17 WG10
+working group.
+
+## ISO 18013-7 Reader Website
+
+The `wwwverifier` module contains the source code for a website acting as an
+mdoc reader according to the latest ISO 18013-7 working draft (as of Sep 2023)
+and it's implementing  the so-called REST API. There is currently a test instance
+of this application available at https://mdoc-reader-external.uc.r.appspot.com/.
+The Wallet Android  application also has support for the REST API and registers
+on Android for the `mdoc://` URI scheme. This can be tested end-to-end by going
+to the reader  website (URL above) and clicking on one of the "Request" buttons,
+and then  hitting the `mdoc://` link presented on the site. This will cause the
+browser  to invoke the Wallet app which will then connect to the reader and send
+the credential after user consent.
+
+# Alignment with the OpenWallet Foundation Mission
+
+The project is already open-source and operating as such. It implements an important
+standard for digital wallets on a widely used mobile platform and is already
+in use in production applications, including Google Wallet.
+
+# Current Code of Conduct
+
+The project is currently using the [Google Code of Conduct](https://github.com/google/identity-credential/blob/master/docs/code-of-conduct.md).
+
+# TAC Sponsor
+--
+
+# Project License
+Apache 2.0
+
+# Source Control
+https://github.com/google/identity-credential
+
+# Issue Tracker
+https://github.com/google/identity-credential/issues
```

Comment:
I see the issue tracker is already quite populated. Are we going to archive and keep these open tickets in their place, or will we transfer them to the new place (repo) under the OWF?

Reply:
We will transfer them, the main scope/direction of the project I think remains the same, we're just finding a new home for it!